lib/constants.ts

```diff
@@ -15,3 +15,5 @@ export const BOOSTER_VERSION_REGEX = /^(\d+.\d+.\d+)-(\d+)(-\w+)?$/;
 export const BOOSTER_NAME_REGEX = /^spring-boot-(.+)-booster$/;
 
 export const UPSTREAM_RELEASE_VERSION_REGEX = /^(\d+.\d+.\d+).RELEASE$/;
+
+export const LICENSES_GENERATOR_PATH = "etc/licenses-generator-shaded.jar";
```

Comment:
I think it would be best if this were configurable via the mechanism we have in: https://github.com/snowdrop/snowdrop-automation-client/tree/master/config.

Also, shouldn't it be `/etc/licenses-generator-shaded.jar`?

Reply:
Hmm, not sure. That etc directory is in the project (https://github.com/snowdrop/snowdrop-automation-client/pull/8/files#diff-bffc0c165e0578c6c9466893867c2b9f), so it shouldn't change

Reply:
Not sure where I saw your comment regarding the missing `/` before `etc`. We can add it or not, it doesn't matter much because this is a relative path in the project. Whenever this constant is used it is used with `resolve(LICENSES_GENERATOR_PATH)` which then give an absolute path to the file.

Reply:
OK, let's leave it as it is and get some experience using it - although I agree that overriding at runtime is not something that would happen to often.
We can even automate the releases of the licenses generator to commit to this project when a new release is made

Reply:
Yeah, I had added the comment about the leading `/` before I saw that you had added the uber-jar to the project - so disregard that comment please :)

Reply:
I say we start as simple as possible :). Furthermore the generator doesn't change all that ofter :P